src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_management_commands.conf.j2

```diff
@@ -2,7 +2,9 @@
 [program:{{ project }}-{{ deploy_env }}-{{ command_name }}]
 directory={{ code_home }}
 environment=TMPDIR="{{ encrypted_tmp }}"
-command={{ py3_virtualenv_home }}/bin/python manage.py {{ command_name }}
+environment={% for name, value in item.env_vars.items() %}{% if value %}{{ name }}="{{ value }}",{% endif %}{% endfor %}
+
+command={{ django_bash_runner if prometheus_monitoring_enabled else django_direct_runner }} {{ command_name }}
```

Comment:
@snopoke this gets applied too celery also. should we remove this since we are planning for pushgateway.

Reply:
It'd only get's applied to these processes which aren't celery: https://github.com/dimagi/commcare-cloud/blob/e1588ea51ca659c438615a89283e32f6c964c7dd/environments/icds-cas/app-processes.yml#L10-L19

Reply:
That's right but here we aren't including the celery server so it's missing django_bash_runner file.
https://github.com/dimagi/commcare-cloud/pull/3796/files#diff-e52ad3f8b83421137281fc6530126e1dR82-R87

Reply:
ok thanks

Reply:
fixed